strawberryfields/backends/bosonicbackend/backend.py

```diff
@@ -383,7 +383,7 @@ def prepare_gaussian_state(self, r, V, modes):
         self.circuit.from_covmat(cov, modes)
         self.circuit.from_mean(means, modes)
 
-    def prepare_cat(self, alpha, phi, representation, cutoff, D):
+    def prepare_cat(self, alpha, phi, representation, amplcutoff, D):
```

Comment:
Note that in the original PR for the `bosonic` backend we use `cutoff` for the small real number used to decide whether to include a Gaussian in phase space. This is inconsistent with the use of `cutoff` for the Fock space truncation thus I've renamed it in the `bosonic` backend.